### ISSUE TITLE
Big vertical sliders (light, cover) don't work in Firefox

### ISSUE THREAD

**Issue by karwosts** [home-assistant/frontend#23345]
### Checklist

- [X] I have updated to the latest available Home Assistant version.
- [X] I have cleared the cache of my browser.
- [X] I have tried a different browser to see if it is related to my browser.
- [ ] I have tried reproducing the issue in [safe mode](https://www.home-assistant.io/blog/2023/11/01/release-202311/#restarting-into-safe-mode) to rule out problems with unsupported custom resources.

### Describe the issue you are experiencing

For as long as I can remember the big sliders in more-info don't work well for me in firefox. 

I click and hold to drag up or down but there is no visual response of anything happening. The percentage indicator does not appear, and the slider does not visually move. 

On releasing the drag the slider does move to the end target, but there's no nice visual feedback like I see in Chrome.

I guess it's something internal to hammerjs but I'm not sure. I didn't find any other open issue about it, so opened this one. Would be curious to hear if its everybody else as well or just me.

### Describe the behavior you expected

Slider is responsive like in chrome.

### Steps to reproduce the issue

1. Open light more info
2. Click and hold on the slider
3. Move mouse up/down
...

### What version of Home Assistant Core has the issue?

2024.12

### In which browser are you experiencing the issue with?

Firefox 133.0.3 (64-bit)

### Which operating system are you using to run this browser?

Win10

**Comment by karwosts** [comment]
still no

**Comment by timmo001** [comment]
Strange, just tried on Firefox 148.0.2 (64-bit) and had no issues

**Comment by karwosts** [comment]
Do you get responsive feedback, like I described? Like it still physically can be changed, but:

> I click and hold to drag up or down but there is no visual response of anything happening. The percentage indicator does not appear, and the slider does not visually move.

Wonder what I'm doing wrong as I've had this issue for years and it's still present...

**Comment by timmo001** [comment]
Yeah the percentage indicator shows, and slides, no visual difference with chromium based browsers 🤷‍♂️

**Comment by karwosts** [comment]
Hmm ok. Will try to think of something else...
